db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/schedule/CronSchedule.java

```diff
@@ -39,19 +40,27 @@ public class CronSchedule implements Schedule, Serializable {
 
   private final String pattern;
   private final ZoneId zoneId;
+  private final CronType cronType;
```

Comment:
I think this should be of the `CronExpressionStyle` as well if there are no problems with that? (since it might get serialized and persisted)

Reply:
Done